metabase/metabase
# Add values source to text tags

Epic https://github.com/metabase/metabase/issues/26898

How to test:
- New -> SQL query -> `SELECT FROM PRODUCTS WHERE CATEGORY = {{category}}`
- `How should users filter on this variable?` -> `Dropdown list` -> `Custom list` -> Enter some values and click `Done`
- Save the question. The feature doesn't work for unsaved questions (yet)
- Click on the filter widget in the editor. It should display your custom values

<img width="1184" alt="Screenshot 2023-01-27 at 18 48 34" src="https://user-images.githubusercontent.com/8542534/215145171-08f491d1-54db-42b7-a972-311f6749139a.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27913)
